client/state/ui/guided-tours/selectors.js

```diff
@@ -74,6 +77,13 @@ const getTourFromQuery = createSelector(
 	getInitialQueryArguments
 );
 
+const DAY_IN_MILLISECONDS = 1000 * 3600 * 24;
```

Comment:
Feels weird to keep this constant at the top of the file, since it has nothing to do with the module in general and is only needed in `isNewUser`. But it feels weird here too, as well as inside the function definition. Which is better?